src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -40,8 +42,10 @@ public void addToCart(@RequestParam Long userId, @RequestParam Long movieSession
     }
```

Comment:
Replace all ?userId @RequestParam in your code with Authentication object.
in row 39